src/rpc/reflect/openrpc_types.rs

```diff
@@ -144,25 +144,27 @@ impl Params {
                 duplicates.join(", ")
             )));
         }
-        if let Some((first_opt_ix, first_opt_param)) =
-            params.iter().enumerate().find(|(_, it)| !it.required)
-        {
-            let late_mandatory_params = params
-                .iter()
-                .enumerate()
-                .filter(|(ix, it)| it.required && *ix > first_opt_ix)
-                .map(|(_, it)| it.name.clone())
-                .collect::<Vec<_>>();
-
-            if !late_mandatory_params.is_empty() {
-                return Err(ParamListError(format!(
-                    "Mandatory parameters may not follow optional parameters, \
-                            but the optional parameter {} is followed by [{}]",
-                    first_opt_param.name,
-                    late_mandatory_params.join(", ")
-                )));
-            }
-        };
+
```

Comment:
What is this piece of commented-out code?

Reply:
I added a comment at the top of the block but it's not obvious. : ) Basically 
this code block panics when mandatory parameters follow optional parameters, but some of the RPC methods violates this rule.

```rust
// `GasEstimateMessageGas::Params` is `(Message, Option<MessageSendSpec>, ApiTipsetKey)
```

Reply:
So what do we do with this? In general, commented-out code should not exist.

Reply:
I'd create a tracking issue and get input from @aatifsyed

Reply:
https://github.com/ChainSafe/forest/issues/4263